cmd/ipfs/daemon.go

```diff
@@ -423,6 +446,35 @@ func serveHTTPGateway(req cmds.Request) (error, <-chan error) {
 	return nil, errc
 }
 
+type listing struct {
+	Uri  string
+	Keys []string
+}
+
+func loadKeySetFromURLs(urls []string) (key.KeySet, error) {
+	ks := key.NewKeySet()
+	for _, url := range urls {
+		resp, err := http.Get(url)
```

Comment:
i think they should work kind of like branches in git.

- there's a remote endpoint with a bunch of named lists (a `tree` of lists if you will)
- fetching them locally tends to use the same name, but can rebind them.